README.md

```diff
@@ -14,7 +14,7 @@ That's it. ArcGIS Runtime SDK for iOS has been integrated into your project as a
 Try out some of the tutorials.
 
 ## Requirements
-- Xcode 12.0 or newer
+- Xcode 13.0 or newer
```

Comment:
We don't need a min of Xcode 13 as far as i know. Any reason you know?
I have been using my Xcode 12.4 for all testing so far and it has worked.

Reply:
Ah whoops sorry I was moving too quickly I was thinking iOS 13